Makefile

```diff
@@ -25,7 +25,7 @@ GOPATH=$(shell go env GOPATH)
 GOOS=$(shell go env GOOS)
 GOBIN=$(shell pwd)/bin
 
-IMAGE?=633339324534.dkr.ecr.us-east-1.amazonaws.com/aws-fsx-openzfs-csi-driver
```

Comment:
nit: PRs shouldn't be changing the image, but it doesn't really matter since this isn't pointing to the public ecr yet

Reply:
Agreed, will fix in next revision.